src/client/store/data/hooks/useOriginalDataPointData.ts

```diff
@@ -1,15 +1,17 @@
-import { NodeValue, TableNames } from '@meta/assessment'
+import { TableNames } from '@meta/assessment'
+import { RecordColumnData } from '@meta/data'
 
 import { useAppSelector } from '@client/store'
 import { useAssessment, useCycle } from '@client/store/assessment'
 import { useCountryIso } from '@client/hooks'
 
-export const useOriginalDataPointData = (): Record<string, Record<string, NodeValue>> | undefined => {
+export const useOriginalDataPointData = (): RecordColumnData | undefined => {
   const assessment = useAssessment()
   const cycle = useCycle()
   const countryIso = useCountryIso()
 
   return useAppSelector(
-    (state) => state.data[assessment.props.name][cycle.name].tableData[countryIso]?.[TableNames.originalDataPointValue]
-  )
+    (state) =>
+      state.data.tableData?.[assessment.props.name]?.[cycle.name]?.[countryIso]?.[TableNames.originalDataPointValue]
```

Comment:
better to add  a util method  to extract data.tableData?.[assessment.props.name]?.[cycle.name]?.[countryIso]?.[TableNames.originalDataPointValue] ? what do you think ?